controls/radcallout/key-properties.md

```diff
@@ -12,11 +12,23 @@ position: 3
 
 This article will list the key properties provided by the RadCallout control.
 
-* __ArrowAnchorPoint__: A property of type __Point__ that gets or sets the anchor point of the arrow geometry. 
+### __Geometry customization__
 
-* __ArrowBasePoint1__: A property of type __Point__ that gets or sets the first base point of the arrow geometry.
+The RadCallout provides 3 properties for customizing the arrow of its geometry - ArrowBasePoint1 , ArrowBasePoint2 and ArrowAnchorPoint.
 
-* __ArrowBasePoint2__: A property of type __Point__ that gets or sets the second base point of the arrow geometry.
+* __ArrowAnchorPoint__: A property of type __Point__ that gets or sets the __anchor point__ of the arrow geometry. 
+* __ArrowBasePoint1__: A property of type __Point__ that gets or sets the __first base point__ of the arrow geometry.
+* __ArrowBasePoint2__: A property of type __Point__ that gets or sets the __second base point__ of the arrow geometry.
+
+They are of type Point and represent relative coordinates (between 0 and 1) where for example 
```

Comment:
"They are of type..". I suggest using the following instead of "they" - "The properties are of type...".

The "topleft point" and "'bottom-right' point".  Use consistent wording. I prefer plain text, so I would say "top left point" and "bottom right" point, but the "top-left" and "bottom-right" sounds good to me too.